fix(labware-library): remove python test protocol affordances from labware-creator

b-cooper opened the pull request:
# Overview

The utility of the python test protocol generated from the labware creator has worn-down and been
replaced by other more helpful trouble shooting tools. Return the labware json file by itself
instead of a zip

Closes [RAUT-198](https://opentrons.atlassian.net/browse/RAUT-198)
Closes RAUT-199
Closes RAUT-200
Closes RAUT-202

# Risk assessment
medium, large chunk of code removal 

[RAUT-198]: https://opentrons.atlassian.net/browse/RAUT-198?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

b-cooper committed:
refactor(labware-library): remove python test protocol affordances from labware-creator

The utility of the python test protocol generated from the labware creator has worn-down and been
replaced by other more helpful trouble shooting tools. Return the labware json file by itself
instead of a zip

Closes RAUT-198

b-cooper committed:
remove unused protocol templates

b-cooper committed:
remove determinMultiChannelSupport util

shlokamin commented:
looks like u gotta update a snapshot test

b-cooper committed:
update snaps

b-cooper committed:
remove unused tests

b-cooper committed:
update cypress tests

jerader reviewed (approved):
smoke tested custom labware creator and it works! Looks like a bunch of cypress tests are failing.

b-cooper committed:
Merge branch 'edge' into labware-creator_remove-test-protocol

b-cooper committed:
fix closure

b-cooper committed:
remove pipettes select tests from cypress

b-cooper committed:
delint

b-cooper committed:
format

b-cooper committed:
Merge branch 'edge' into labware-creator_remove-test-protocol

b-cooper committed:
resolve conflicts

b-cooper committed:
fix up merged test failures

b-cooper committed:
update snapshot

b-cooper committed:
lint

SyntaxColoring commented:
Hooray!

Might want to call this a `feat` or `fix` instead of a `refactor`, for the sake of changelog scanning.

b-cooper committed:
update file comparison

b-cooper committed:
fix up file blob usage

b-cooper committed:
fix(labware-library): remove python test protocol affordances from labware-creator

Remove the python test protocol generated with custom labware output from the labware creator. There
are more helpful troubleshooting tools elsewhere in the ecosystem. Return only the JSON file instead
of a zip archive

Closes RAUT-198, Closes RAUT-199, Closes RAUT-200, Closes RAUT-202

b-cooper committed:
remove accidental top level cypress
